### PR TITLE
fix: copy plain text

### PR DESCRIPTION
**Pull request by JulianWielga**
fix: selection actions
refactor: extract selection actions & keyboard shortcuts
feat: select all
feat: enable paste button only on valid clipboard contents
fix: translations

### CODE REVIEW

File: ui/client/components/graph/SelectionContextProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        return
+      }
+
+      if (canModifySelected) {
+        await ClipboardUtils.writeText(JSON.stringify(selection))
+        const {nodes} = selection
+        if (!silent) {
+          dispatch(success(t("userActions.copy", {
+            defaultValue: "Copied node",
+            defaultValue_plural: "Copied {{count}} nodes",
```
Comment (lciolecki) [inline ui/client/components/graph/SelectionContextProvider.tsx:116]: Two styles of formatting..? Maybe defaultValuePlural?

File: ui/client/containers/Auth/StrategyInitializer.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ export function StrategyInitializer(props: PropsWithChildren<Props>): JSX.Elemen
           },
           buttonLabel: t("auth.StrategyInitializer.errors.504.buttonLabel", "Go to authentication page"),
           description: t(
-            "auth.StrategyInitializer.errors.504.description",
+            "auth.StrategyInitializer.errors.1024.description",
```
Comment (lciolecki) [inline ui/client/containers/Auth/StrategyInitializer.tsx:87]: Why did you change it?

**Review by lciolecki — APPROVED** [review]
LGTM, but please look at my comments.

File: ui/client/components/graph/SelectionContextProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+  const [state, setState] = useState<"denied" | "granted" | "prompt">()
+  const [text, setText] = useState("")
+  const [content, setContent] = useState("")
+
+  const parse = useClipboardParse()
+
+  if (state === "granted") {
+    navigator.clipboard.readText().then(text => {
+      setText(text)
+    }).catch(() => {return})
```
[the remark this answers is not in the document]
Comment (JulianWielga) [reply ui/client/components/graph/SelectionContextProvider.tsx:58]: just ignore possible error - it's expected sometimes

File: ui/client/components/graph/SelectionContextProvider.tsx [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+          NodeUtils.isAvailable(node, processDefinitionData, processCategory))
+      return isValid ? selection : null
+    },
+    [processCategory, processDefinitionData],
+  )
+}
+
+function useClipboardPermission(): boolean | string {
+  const clipboardPermission = useRef<PermissionStatus>()
+  const [state, setState] = useState<"denied" | "granted" | "prompt">()
```
[the remark this answers is not in the document]
Comment (JulianWielga) [reply ui/client/components/graph/SelectionContextProvider.tsx:49]: brings nothing except boilerplate

File: ui/client/components/graph/SelectionContextProvider.tsx (hunk 1 above)
Comment (JulianWielga) [reply ui/client/components/graph/SelectionContextProvider.tsx:116]: what? please read about i18next.

File: ui/client/containers/Auth/StrategyInitializer.tsx (hunk 2 above)
Comment (JulianWielga) [reply ui/client/containers/Auth/StrategyInitializer.tsx:87]: `ACCESS_TOKEN_CODE` is `1024` not `504` which is `HTTP_TIMEOUT_CODE`
there was two translation keys with different default value. this mistake appeared when changed extract method.
